compiler/tests/fail/register_allocation/x86-64/bug_1026.jazz

```diff
@@ -0,0 +1,10 @@
+export fn main(reg u32 x) -> reg u32 {
```

Comment:
I don't understand the interest of the test.
I mean we don't check that printing will not change, so what are we testing here ?

Reply:
You are right. This is just an example that I’ve been using to check these changes. However, there is no need to keep in in the repository. Let me remove it.

Reply:
Done.